src/test/java/io/anserini/eval/RelevanceJudgmentsTest.java

```diff
@@ -35,7 +35,7 @@ public int getQrelsCount(RelevanceJudgments qrels) throws IOException{
 
   @Test
   public void testRobust04() throws IOException{
-    RelevanceJudgments qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.robust04.txt");
+    RelevanceJudgments qrels = new RelevanceJudgments("tools/topics-and-qrels/qrels.robust04.txt");
```

Comment:
Wait, how are tests passing now?

Reply:
Previously, all topics were downloaded to the cache before it was searched locally.
After the fix, files are searched locally first and then download from the web if not found.